UM-USElab/gradecraft-development
# Grade scheme elements refactor

### Status
IN PROGRESS

### Description
In anticipation for some upcoming features requested for the Grade Scheme Elements page, we should look at tidying up the current implementation so that it's more consistent and readable with the rest of our more recent Angular work.

### Todos

### Gem dependencies
N/A

### Migrations
NO

### Steps to Test or Reproduce
- Validation occurs onblur, so any conflicts should appear when focus on the current input grade input is lost. Ensure that if there is not a zero threshold, one is automatically added to the form.
- Verify that GradeSchemeElements are being saved correctly and that errors are presented as expected.

### Impacted Areas in Application
Edit GradeSchemeElements

======================
Closes #3030
